src/components/TokenClaimModal.tsx

```diff
@@ -111,6 +118,23 @@ export default function TokenClaimModal(): ReactElement {
               )}
             </>
           )}
+          {!isClaimableNetwork && (
+            <p style={{ whiteSpace: "pre-line" }}>
+              <Trans i18nKey="disableRewardContent">
+                SDL is currently only deployed on Ethereum Mainnet and is not
```

Comment:
It doesn't seem right to have the text content in the component and the translation file itself.

Reply:
I think, this is the ussage of <Trans> component. Because it is having <Link/> component, I have used <Trans/> component instead of useTranslation.

Reply:
Good to know, thanks for the link.